js/components/spectrogram.js

```
@@ -0,0 +1,943 @@
+   * Reinitializes the spectrogram plugin if it has not been initialized.
+   *
+   * Checks if a global wavesurfer instance exists and the spectrogram is not already set up.
+   * If both conditions are met, it initializes the spectrogram using the configured maximum height
+   * and registers it as a plugin with wavesurfer.
+   */
+  reInitSpec() {
+    const wavesurfer = this.wavesurfer;
+    const spectrogram = this.spectrogram;
+    const config = this.getConfig();
+    if (wavesurfer && !spectrogram) {
+      spectrogram = this.initSpectrogram(config.specMaxHeight);
```

Biome (1.9.4)

[error] Can't assign spectrogram because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)